### PR TITLE
[EDU-4448] feat: enforce HLS caching guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4448]

### Changes

- Add enforce HLS caching guide.
- Refactor Live Streaming arch and link to guide.
- Add the link to the guides page.
- Add button to Rules Engine reference.

### Additional links

n/a.

[EDU-4448]: https://aziontech.atlassian.net/browse/EDU-4448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+---
+
+import Button from '~/components/Button.astro';
+import Tabs from '~/components/tabs/Tabs'
+
+:::caution[important]
+Azion has two user interfaces: Real-Time Manager and Console, which is in Preview stage. Currently, Console is only available for Developer plans and new accounts. Follow the steps according to the user interface you're using.
+:::
+
+Azion Edge Platform allows you to deliver live streaming content in **HLS format**, being able to easily self-provision and configure your cache policies. This guide covers the step-by-step to enforce HLS cache, managing chunks and playlists caching and setting up rules engines rules.
```
Comment (mrmessa) [inline src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx:16]: *Maybe* linking keywords in the first paragraph would be better than creating a button to use case page.

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+---
+
+import Button from '~/components/Button.astro';
+import Tabs from '~/components/tabs/Tabs'
+
+:::caution[important]
+Azion has two user interfaces: Real-Time Manager and Console, which is in Preview stage. Currently, Console is only available for Developer plans and new accounts. Follow the steps according to the user interface you're using.
+:::
+
+Azion Edge Platform allows you to [deliver live streaming content in HLS format](/en/documentation/architecture/live-streaming-delivery/live-streaming-delivery-with-hls/), being able to easily self-provision and configure your cache policies. This guide covers the step-by-step to enforce HLS cache, managing chunks and playlists caching and setting up rules engines rules.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx:16]:
```suggestion
Azion Edge Platform allows you to [deliver live streaming content in HLS format](/en/documentation/architecture/live-streaming-delivery/live-streaming-delivery-with-hls/), being able to easily self-provision and configure your cache policies. This guide covers the step-by-step to enforce HLS cache, managing chunks and playlists caching, and setting up rules engines rules.
```

File: src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    <Fragment slot="tab.cli">CLI </Fragment>
+    <Fragment slot="tab.console">Console </Fragment>
+    <Fragment slot="tab.rtm">Real-Time Manager </Fragment>
+
+<Fragment slot="panel.cli">
+1. Open [Azion CLI](/en/documentation/products/azion-cli/overview/) in your terminal.
+2. Get the details of an existing edge application using the [list command](/en/documentation/devtools/cli/list/): `azion list edge-application --details`
+3. Enable Tiered Cache for your application: `$ azion update edge-application --application-id 1234 --l2-caching true`
+4. Create an origin: `$ azion create origin --application-id 1234 --name "origin-edge" --origin-type single_origin --addresses "example.com" --host-header "example.com"`
+5. Configure the cache policy for the chunks: `$ azion create cache-setting --application-id 1234 --name "chunks-policy" --browser-cache-settings "override" --browser-cache-settings-maximum-ttl 0 --cdn-cache-settings "override" --cnd-cache-settings-maximum-ttl 60`
```
[the remark this answers is not in the document]
Comment (gabriel-azion) [reply src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx:48]: lembro de ter visto isso alguma vez, posse que o certo seja o errado

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx [hunk 4]
```diff
@@ -0,0 +1,245 @@
+---
+title: How to enforce HLS cache for live streaming delivery
+description: This guide covers the step-by-step to enforce HLS cache, managing chunks and playlists caching and setting up Rules Engine rules.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx:3]:
```suggestion
description: This guide covers the step-by-step to enforcing HLS cache, managing chunks and playlists caching, and setting up Rules Engine rules.
```

File: src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MarianaAguilera) [reply src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx:48]: testei e funcionou com esse que está no guia, sim!

**Review by mrmessa — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 5]
```diff
@@ -0,0 +1,249 @@
+---
+title: Como impor cache HLS para entrega de streaming ao vivo
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:2]:
```suggestion
title: Como implementar cache HLS para entrega de streaming ao vivo
```

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+description: Este guia abrange o passo a passo para impor cache HLS, gerenciando o cache de chunks e playlists e configurando regras no Rules Engine.
+meta_tags: architecture, content delivery, hls, edge computing
+namespace: docs_guides_enforce_hls_cache
+permalink: /documentacao/produtos/guias/impor-cache-hls/
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:6]:
```suggestion
permalink: /documentacao/produtos/guias/implementar-cache-hls/
```

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+---
+
+import Button from '~/components/Button.astro';
+import Tabs from '~/components/tabs/Tabs'
+
+:::caution[importante]
+A Azion possui duas interfaces de usuário: o Real-Time Manager (RTM) e o Console, que está em fase de Preview. Atualmente, o Console está disponível apenas para planos Developer e novas contas. Siga os passos de acordo com a interface de usuário que você está usando.
+:::
+
+A Plataforma de Edge da Azion permite que você [entregue conteúdo de streaming ao vivo no formato HLS](/pt-br/documentacao/arquitetura/live-streaming-delivery/live-streaming-delivery-com-hls/), capacitando-lhe a autoprovisionar e configurar suas política de cache facilmente. Este guia abrange o passo a passo para impor o cache HLS, gerenciando o cache de chunks e playlists e configurando regras no Rules Engine.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:16]:
```suggestion
A Plataforma de Edge da Azion permite que você [entregue conteúdo de streaming ao vivo no formato HLS](/pt-br/documentacao/arquitetura/live-streaming-delivery/live-streaming-delivery-com-hls/), capacitando-lhe a autoprovisionar e configurar suas política de cache facilmente. Este guia abrange o passo a passo para implementar o cache HLS, gerenciando o cache de chunks e playlists e configurando regras no Rules Engine.
```

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+
+## Pré-requisitos
+
+- Uma edge application existente na Azion, ou [crie uma nova](/pt-br/documentacao/produtos/guias/build/criar-uma-aplicacao/).
+- [Um domínio associado à edge application](/pt-br/documentacao/produtos/guias/configurar-dominio/).
+- [Tiered Cache habilitado](/pt-br/documentacao/produtos/guias/billing-and-subscriptions/) em sua conta.
+
+---
+
+## Imponha políticas de cache HLS
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:28]:
```suggestion
## Implemente políticas de cache HLS
```

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+
+Para impor políticas de cache HLS na edge application, siga os passos explicados embaixo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:30]:
```suggestion
Para implementar políticas de cache HLS na edge application, siga os passos explicados embaixo.
```

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+1. Clique no botão **New Rule** button e selecione a opção **Request Phase**.
+2. Dê um nome e uma descrição (opcional) à sua regra.
+3. Em **Criteria**, defina `if ${uri}` *matches* `\*.m3u8`.
+4. Em **Behaviors**, selecione **Set Cache Policy** e adicione a política para a playlist que você criou no passo anterior.
+5. Clique no botão **Save**.
+</Fragment>
+
+</Tabs>
+
+Pronto! Agora você pode configurar sua fonte e codificador apontando para a Azion e transmitir seu conteúdo, impondo o cache HLS.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:237]:
```suggestion
Pronto! Agora você pode configurar sua fonte e codificador apontando para a Azion e transmitir seu conteúdo, implementando o cache HLS.
```

File: src/content/docs/pt-br/pages/guias/guides.mdx [hunk 11]
```diff
@@ -29,6 +29,7 @@ permalink: /documentacao/produtos/guias/
 - [Como usar um bucket do Edge Storage como origem de uma edge application estática](/pt-br/documentacao/produtos/guias/usar-bucket-como-origem/)
 - [Como configurar portas HTTP e HTTPS para origens e endereço de entrega](/pt-br/documentacao/produtos/guias/configurar-portas/)
 - [Como acessar o Edge Storage usando o protocolo S3](/pt-br/documentacao/produtos/guias/protocolo-s3-para-edge-storage/)
+- [Como impor cache HLS para entrega de streaming ao vivo](/pt-br/documentacao/produtos/guias/impor-cache-hls/)
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/guides.mdx:32]:
```suggestion
- [Como implementar cache HLS para entrega de streaming ao vivo](/pt-br/documentacao/produtos/guias/implementar-cache-hls/)
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/rules-engine-edge-app.mdx [hunk 12]
```diff
@@ -317,6 +317,8 @@ Este behavior é incluído automaticamente pela Azion toda vez que você selecio
 
 A política de cache da Azion para transmissões live em HLS é de 5 segundos de cache para playlists (`.m3u8`) e 60 segundos de cache para chunks (`.ts`).
 
+<Button href="/pt-br/documentacao/produtos/guias/impor-cache-hls/" text="consulte o guia para impor cache HLS" variant="secondary"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/rules-engine-edge-app.mdx:320]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/implementar-cache-hls/" text="consulte o guia para implementar cache HLS" variant="secondary"></Button>
```

File: src/content/docs/pt-br/pages/arquiteturas/live-streaming-delivery/live-streming-delivery-com-hls.mdx [hunk 13]
```diff
@@ -42,20 +42,7 @@ Esta solução é ideal para provedores de conteúdo que necessitam de alta fide
...
-        - Na seção **Advanced Cache Key**, defina o comportamento de sua aplicação em relação à segmentação de objetos em cache. A recomendação é selecionar **Content does not vary**, para **Query String** e/ou **Cookies**, de acordo com suas necessidades.
-    - Em seguida, crie uma política de cache para a playlist.
-        - Repita o processo anterior, mas desta vez defina um **Maximum TTL** de `0` para **Browser Cache Settings** e `5` para **Edge Cache Settings**. Defina também as configurações da **Advanced Cache Key**.
-    - Ainda na página da aplicação, abra a aba **Rules Engine**:
-        - Crie uma regra para os chunks: selecione `if ${uri}` *matches* `.\*.ts` como critério (criteria) e **Set Cache Policy** como comportamento (behavior), adicionando a política para os chunks criada no passo anterior.
-        - Crie uma regra para a playlist: defina `if ${uri}` *matches* `\*.m3u8` como critério (criteria) e **Set Cache Policy** como comportamento (behavior), adicionando a política para a playlist criada no passo anterior.
+3. Configure [políticas de cache para HLS](/pt-br/documentacao/produtos/guias/impor-cache-hls/) na edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/live-streaming-delivery/live-streming-delivery-com-hls.mdx:45]:
```suggestion
3. Configure [políticas de cache para HLS](/pt-br/documentacao/produtos/guias/implementar-cache-hls/) na edge application.
```

File: src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx [hunk 14]
```diff
@@ -0,0 +1,249 @@
+---
+title: Como implementar cache HLS para entrega de streaming ao vivo
+description: Este guia abrange o passo a passo para impor cache HLS, gerenciando o cache de chunks e playlists e configurando regras no Rules Engine.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/edge-application/ea-enforce-hls.mdx:3]:
```suggestion
description: Este guia abrange o passo a passo para implementar cache HLS, gerenciando o cache de chunks e playlists e configurando regras no Rules Engine.
```

**Pull request merged**